backend/doc/apache.md

```diff
@@ -84,6 +84,18 @@ Enable this configuration file:
 
     sudo ln -s /etc/apache2/sites-available/tracim.conf /etc/apache2/sites-enabled/tracim.conf
 
+#### Permit access to guest download with old direct link url used before Tracim 3.x (old API path is `/api/v2/`)
+
+example:
+- old url (Tracim < 3.x): `http://localhost/api/v2/public/guest-download/.....`
+- new url (Tracim ≥ 3.x): `http://localhost/api/public/guest-download/.....`
+
+You just need to add this in your apache2 configuration to make sure old direct link working correctly:
```

Comment:
I would avoid using words like "just", "easily", "simply" ... in documentations since, in IT, nothing actually ever is

Reply:
I would agree with @Skylsmoi

Reply:
thanks for this information.

Reply:
So `You may` or `You can` ?

Reply:
I choose `You can`.